src/rate.rs

```diff
@@ -229,7 +229,7 @@ pub(crate) fn bexp64(logq57: i64) -> i64 {
 // input: a 64-bit integer in Q0 (no fraction).
 // output: a 64-bit log in Q57.
 // TODO: Mark const once we can use local variables in a const function.
-fn blog64(w: i64) -> i64 {
+pub fn blog64(w: i64) -> i64 {
```

Comment:
note to self, once it lands refactor the maths function from rate.rs to a stand alone file.